src/uct/api/uct_def.h

```diff
@@ -111,6 +117,13 @@ typedef int                      uct_worker_cb_id_t;
     |<---- stride ----->|<---- stride ----->|
    @endverbatim
  *
+ * In order to describe an interleaved memory layout, each structure contains
```

Comment:
I want to see how we can make the simplest possible API,
we can also add "repeat_count" to ep_mem_reg, and drop "count" and "ilv_ratio" from uct_iov_t (keep only buffer, key, len, stride)
Therefore, ABBBCCABBBCCABBBCC could be represented as follows:
```
iov[0].ptr = &A
iov[0].length=1
iov[0].memh= key_A
iov[1].ptr = &B
iov[1].length=3
iov[1].memh= key_B
iov[2].ptr = &C
iov[2].length=2
iov[2].memh= key_C
ep_mem_reg(iov, iovcnt=3, repeat_count=3)
```